lms/static/js/my_courses_dropdown.js

```diff
@@ -1,81 +1,87 @@
-$(document).ready(function() {
+(function(require) {
     'use strict';
 
-  // define variables for code legibility
-    var dropdownMenuToggle = $('.dropdown');
-    var dropdownMenu = $('.dropdown-menu');
-    var menuItems = dropdownMenu.find('a');
+    require([
+        'jquery',
+        'edx-ui-toolkit/js/utils/constants'
+    ], function($, constants) {
+         // define variables for code legibility
+        var $dropdownMenuToggle = $('.dropdown');
+        var $dropdownMenu = $('.dropdown-menu');
+        var menuItems = $dropdownMenu.find('a');
```

Comment:
It'd be great to use a classname here, instead of an element, especially if we convert these to buttons in the near future.